db/migrate/20160302203922_remove_is_datacenter_from_ems_folder.rb

```diff
@@ -0,0 +1,22 @@
+class RemoveIsDatacenterFromEmsFolder < ActiveRecord::Migration[5.0]
+  class EmsFolder < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    # Go through all existing EmsFolder records and set :type => Datacenter
+    # if the is_datacenter column is true before deleting it
+    EmsFolder.all.each do |folder|
+      folder.update_attributes!(:type => "Datacenter") if folder.is_datacenter?
+    end
+
+    remove_column :ems_folders, :is_datacenter
+  end
+
+  def down
+    add_column :ems_folders, :is_datacenter, :boolean
+    EmsFolder.all.each do |folder|
```

Comment:
@jrafanie how does this look? https://github.com/ManageIQ/manageiq/pull/7070/commits/09b8555a43042bb47621a33740fc6e0bc26253b3